modulefiles/chgres_cube.wcoss_cray

```diff
@@ -7,20 +7,23 @@ module rm intel
 module load intel/16.3.210
 module load cray-mpich/7.2.0
 module load craype-haswell
-module load cray-netcdf
+module load cray-netcdf/4.3.3.1
+module load cray-hdf5/1.8.14
 module load w3nco-intel/2.0.6
 module load nemsio-intel/2.2.3
 module load bacio-intel/2.0.2
 module load sp-intel/2.0.2
 module load sigio-intel/2.1.0
 module load sfcio-intel/1.0.0
 
-# module use /gpfs/hps3/emc/nems/noscrub/emc.nemspara/soft/modulefiles
-# module load esmf/7.1.0r
-export ESMFMKFILE=/gpfs/hps3/emc/global/noscrub/George.Gayno/esmf/8_0_0_bs20/lib/esmf.mk
+module use /gpfs/hps3/emc/nems/noscrub/emc.nemspara/soft/modulefiles
+module load esmf/8.0.0
+module rm gcc
+module load gcc/4.9.2
 
-export WGRIB2API_INC=/gpfs/hps3/emc/meso/save/Dusan.Jovic/wgrib2/include
-export WGRIB2_LIB=/gpfs/hps3/emc/meso/save/Dusan.Jovic/wgrib2/lib/libwgrib2.a
+
+export WGRIB2API_INC=/gpfs/hps3/emc/global/noscrub/George.Gayno/wgrib2/include
+export WGRIB2_LIB=/gpfs/hps3/emc/global/noscrub/George.Gayno/wgrib2/lib/libwgrib2.a
```

Comment:
Will these be more officially located at some point (be in a non-personal space)?

Reply:
And on Dell, I point to Wesley's home directory for this library.

I will keep pushing the NCEPLIBS team to formally implement it.  However, since chgres_cube is not used in operations yet, pointing to personal space should be ok for now.

Unfortunately, the community partners who wanted grib2 capability in chgres_cube made the decision to use wgrib2 instead of g2lib.  I would have chosen the latter, which is more flexible and formally supported.

Reply:
Gotcha, thanks!